JitsiConference.js

```diff
@@ -3596,6 +3598,44 @@ JitsiConference.prototype._stopP2PSession = function(options = {}) {
     }
 };
 
+/**
+ * Updates room presence if needed and send the packet in case of a modification.
+ * @param {JingleSessionPC} jingleSession the session firing the event, contains the peer connection which
+ * tracks we will check.
+ * @param {Object|null} ctx a context object we can distinguish multiple calls of the same pass of updating tracks.
+ */
+JitsiConference.prototype._updateRoomPresence = function(jingleSession, ctx) {
+    if (ctx && ctx.skip) {
+        ctx.skip = true;
```

Comment:
is it not possible to skip the call in the place which sets the `ctx`?

Reply:
actually I don't see it being set anywhere?

Reply:
https://github.com/jitsi/lib-jitsi-meet/pull/1814/files#diff-2ab65ee8ec5b7568335948135d9d2a5b7010c054a1d821f9774a7b542689b282R2525

Reply:
It is just the case if you have a source add and source remove in one call of update ssrc to send one presence, not two.

Reply:
but there it only defines the `ctx` object, where is the `ctx.skip` being set?

Reply:
It is set here, once it passes this method and will skip the second invocation.

Reply:
Oh, I need to set it after the if 🤦‍♂️

Reply:
updated.

Reply:
I'm not a big fan of this solution, but should do for now... source-remove/source-add should be merged into a single notification.